src/CSV/Import.elm

```diff
@@ -15,7 +15,7 @@ elements.
 
 import Csv
 import Dict
-import Localized
+import Localized exposing (..)
```

Comment:
I really appreciate the changes from `String` to named type alias! Please keep the named import though: again `Localized.Module` is easier to read and understand than just `Module`